next.config.js

```diff
@@ -44,6 +44,8 @@ const nextConfig = {
       'pg-native': false,
     };
 
+    // Exclude pg-cloudflare from client-side bundles 
+    if (!isServer) { config.externals.push('pg-cloudflare'); }
```

Comment:
Without this, we would get an error whenever RepoList is shown (if it called the repos/getReposFromNames function instead of implementing itself with the `conn` Pool received from its parent (`u.tsx` or `repo.tsx`).

Reply:
cool.